Sources/NostrSDK/Filter.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  Filter.swift
+//  
+//
+//  Created by Joel Klabo on 5/26/23.
+//
+
+import Foundation
+
+struct Filter {
+    let ids: [String]?
+    let authors: [String]?
+    let kinds: [Int]?
+    let events: [String]?
+    let pubkeys: [String]?
+    let since: Int?
+    let until: Int?
+    let limit: Int?
+
+    private enum CodingKeys: String, CodingKey {
+        case ids = "ids"
+        case authors = "authors"
+        case kinds = "kinds"
+        case events = "#e"
+        case pubkeys = "#p"
+        case since = "since"
+        case until = "until"
+        case limit = "limit"
+    }
+}
+
+extension Filter: Codable {
```

Comment:
Did you need to implement the Codable requirements? Usually Swift can synthesize these for a simple struct.

Reply:
Good catch, you're right I don't: 6fa6eb7